Exercise_D and E/index.html

```diff
@@ -16,7 +16,54 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <div style="margin-left: 30px">
```

Comment:
Use classes or id's and style using your stylesheet. This keeps your HTML code clean